osu.Game/OsuGame.cs

```diff
@@ -757,7 +765,7 @@ protected override void LoadComplete()
             loadComponentSingleFile(channelManager = new ChannelManager(), AddInternal, true);
             loadComponentSingleFile(chatOverlay = new ChatOverlay(), overlayContent.Add, true);
             loadComponentSingleFile(new MessageNotifier(), AddInternal, true);
-            loadComponentSingleFile(Settings = new SettingsOverlay { GetToolbarHeight = () => ToolbarOffset }, leftFloatingOverlayContent.Add, true);
+            loadComponentSingleFile(Settings = new SettingsOverlay(), leftFloatingOverlayContent.Add, true);
```

Comment:
While I'm ok with making `ToolbarOffset` private and only handling that stuff at game-level once instead of locally in each overlay, that seems a little orthogonal to what this pull is actually claiming to be doing in the title (it's a pure code quality change that doesn't seem have anything to do with implementing the behaviour proposed in any way).

Reply:
This was a remnant of the opposite behavior I was trying to fix. I think it's still good to leave as it fixes the draw size of floating overlays.